src/Features/Core/Portable/LanguageServices/SymbolDisplayService/AbstractSymbolDisplayService.AnonymousTypes.cs

```diff
@@ -33,12 +33,10 @@ where part.Symbol.IsAnonymousType() || part.Symbol.IsTupleType()
                     firstSymbol, directStructuralTypes.ToImmutableArrayOrEmpty(), _semanticModel, _position);
 
                 if (info.TypesParts.Count > 0)
-                {
                     AddToGroup(SymbolDescriptionGroups.StructuralTypes, info.TypesParts);
 
-                    foreach (var (group, parts) in _groupMap.ToArray())
-                        _groupMap[group] = info.ReplaceStructuralTypes(parts, _semanticModel, _position);
-                }
+                foreach (var (group, parts) in _groupMap.ToArray())
+                    _groupMap[group] = info.ReplaceStructuralTypes(parts, _semanticModel, _position);
```

Comment:
was this change needed as well?  thanks!

Reply:
Yes. If you don't do that, you will get this (first example in the issue):
```cs
Foo(default);

void Foo(<tuple> foo)
{
}
```